### ISSUE TITLE
Voice over is not announcing the role of "Upload photo" button.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-iOS-SDK#697]
Category: Screen Reader

MAS Rule: MAS4.2.1

Repro Steps

1. Launch Embedded Social Test App.
2. Navigate to Hamburger button and double tap.
3. Navigate to Sign In link and double tap.
4. Sign In with any (Google, MS, Facebook, Twitter) valid credential and double tap on Sign Up button.
5. Create profile page will be appear.
6. Navigate to the upload photo button via swipe gesture.
7. Verify that voice over is announcing the role or not.

Expected:
Voice over should announce the role of Upload photo button so that screen reader user get correct information.

Actual:
Voice over is not announcing the role of Upload photo button. It is just announcing Upload photo.

Note: 
1. Role is also not defined for Trending topics under the search page.
2. Same issue is repro for people profile on search page.

Test Environment:
OS: iOS 11.0.2
App: Embedded Social Test
App Version: 0.7.7 (7)
Tools: Voice Over, Keyboard
